pipelines/tasks/picard.wdl

```diff
@@ -81,7 +81,7 @@ task CollectDuplicationMetrics {
   String output_filename
 
   command {
-    java -Xmx6g -jar /usr/picard/picard.jar  MarkDuplicates \
+    java -Xmx6g -XX:ParallelGCThreads=2 -jar /usr/picard/picard.jar  MarkDuplicates \
```

Comment:
Did this speed things up?

Reply:
This would be handy for greenteam if it did!  Maybe more importantly did it make it cheaper?